logstash-core/lib/logstash/pipeline.rb

```diff
@@ -176,8 +179,10 @@ def stopped?
   end
 
   def start_workers
-    @inflight_batches = {}
-
+    @filter_queue_client.set_events_metric(metric.namespace([:stats, :events]))
```

Comment:
Well, why not change it to `set_*` if we change the signature. I don't see why we should write non-idiomatic ruby because of a (very unlikely IMHO) future possibility.